### DISCUSSION TITLE
Direct connection child to parent Node

### DISCUSSION THREAD

**Discussion by allangnutzmans** [bcakmakoglu/vue-flow#1716] [Q&A] [answered]
Is there any way to connect the internal node with the external directly?
Current behavior:
![image](https://github.com/user-attachments/assets/7b835db2-d8cc-4ab6-8168-b0d04e6f8c53)

Desired behavior:
![image](https://github.com/user-attachments/assets/7e2f2031-c9c8-46c9-910a-9a052d693622)

**Comment by bcakmakoglu** [comment, marked as answer]
Yeah, but you'll have to create a custom edge and flip source/target positions of the handle so they would lead inwards instead of outwards.
So if a groups handle is at the bottom, flip it and tell `getBezierEdge` to use `targetPosition: Position.Top` instead (same for source handle and sourceposition, flip it).
